python/gsw/main.py

```diff
@@ -0,0 +1,370 @@
+from flask import Flask, request, jsonify
+from flasgger import Swagger
+from flasgger.utils import swag_from
+from elasticsearch import Elasticsearch
+from datetime import datetime
+import threading
+import json
+import imaplib
+import base64
+import os
+import email
+import time
+import logging
+
+app = Flask(__name__)
+app.config["SWAGGER"]={"title": "PAN Ground Software", "uiversion": 2}
+ 
+class read_iridium(object):
+    def __init__(self, radio_keys_config, server_keys_config):
```

Comment:
The number of configs have grown a lot in the past couple of weeks. Can you add a README to the config folder explaining the purpose of the config files?